Mixpanel/Mixpanel.m

```diff
@@ -154,6 +154,25 @@ - (void)setValidationEnabled:(BOOL)validationEnabled {
 }
 #endif
 
+- (BOOL)shouldManageNetworkActivityIndicator {
+    return self.network.shouldManageNetworkActivityIndicator;
+}
+
+- (void)setShouldManageNetworkActivityIndicator:(BOOL)shouldManageNetworkActivityIndicator {
+    self.network.shouldManageNetworkActivityIndicator = shouldManageNetworkActivityIndicator;
+}
+
+- (BOOL)useIPAddressForGeoLocation {
+    return _useIPAddressForGeoLocation;
```

Comment:
I could move the entirety of that to MPNetwork and get rid of it. lemme know if that sounds good